frontend/src/components/ViewTrips/trip.js

```diff
@@ -32,12 +45,11 @@ export function moveCurUserEmailToFront(collaboratorEmailArr) {
  * when trips are added and/or editted. Thus, no error checking is done here
  * on the 'display' side.
  *
- * @param {Object} props These are the props for this component:
- * - tripData: Object holding a Trip document fields and corresponding values.
- * - tripId: Document ID for the current Trip document.
- * - handleEditTrip: Handler that displays the edit trip modal.
- * - key: Special React attribute that ensures a new Trip instance is
- *        created whenever this key is updated
+ * @property {Object} props These are the props for this component:
+ * @property {TripData} props.tripData Object holding a Trip document data.
```

Comment:
Is trip an object type that should have backticks surrounding it?

Reply:
Nope. It is a (firestore) document.